package.json

```diff
@@ -21,6 +22,7 @@
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "@types/react-router-dom": "^5.3.3",
```

Comment:
You don't need to install the types of react-router-dom

It comes with its own type declarations.

I think you can safely remove this package.

Reply:
@ISHIMWE-Aime version 5 and above of react-router-dom has its own type declarations.